scripts/backend_tests.py

```diff
@@ -198,7 +203,7 @@ def _execute_tasks(tasks, batch_size=24):
         log('----------------------------------------')
 
 
-def _get_all_test_targets(test_path=None):
+def _get_all_test_targets(test_path=None, include_load_tests=False):
```

Comment:
include_load_tests=True should be the default. We should update .travis.yml to pass --exclude_load_tests explicitly as an argument, but the default set of tests that runs on the user's machine should include load tests by default.